app/src/main/res/layout/activity_main.xml

```diff
@@ -1,123 +1,116 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView
+<android.support.design.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
     android:background="@drawable/dt"
+    android:orientation="vertical"

```

Comment:
Ah! Thank you very much for reviewing my changes @kkdroidgit ! I missed removing that attribute after replacing the initial **ScrollView** ViewGroup with **CoordinatorLayout** :sweat_smile: That can be removed.
I'm not sure if I should fix this now with a commit (which I believe would automatically update my PR after push) or do we take care of this in a new commit/PR as it would be a minor change and it is not affecting the UI? 
What would you suggest?

Reply:
No problem @amy6 , I'll remove it in my local repo and check it.

Reply:
Totally up to you! Though there shouldn't be any effect on the UI.